### PR TITLE
Add Better Stonewood Path

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Add mod, serves as a part of load after order for several later mods. That's all it does for now.

### CODE REVIEW

**Comment by sibir-ine** [comment]
If you want this mod to load after several others, you should add the rules in this PR. For instance, you can add something like:
```yaml
  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
    url: [ 'http://example.com/' ]
    after:
      - 'Mod1.esp'
      - 'Mod2.esp'
      - 'Mod3.esp'
```
if you want it to load after **Mod1.esp**, **Mod2.esp**, & **Mod3.esp**.

**Comment by IllusiveMan196** [comment]
It loads before them (example: Beautiful Cities needs to load after this mod or else landscape will be broken in at least one part of Balmora) so I'm adding this first

**Comment by sibir-ine** [comment]
Okay, then you just need to add load after rules when adding those mods. You don't need to add this line for them to work.

**Comment by IllusiveMan196** [comment]
3 PRs incoming :) right after this one (still would keep this mod in the list)

**Comment by sibir-ine** [comment]
Do you have any other load after rules, a group assignment, or other data that can be added to it? Otherwise, we don't add entries without any data or with only a URL (unless it's in a regex for other entries in the masterlist that do contain data).

**Comment by IllusiveMan196** [comment]
Seems like yes, a moment please

**Comment by MacSplody** [comment]
This is the mod? [The Stonewood pass reworked](https://www.nexusmods.com/morrowind/mods/49464/)

> Lord order
> Taking into account all the conflicting mods that I know of, this is the load order:
> 
> DD_Caldera_Expansion.esp
> Foyada Mamaea Overhaul.ESP
> Better Landscapes Stonewood Pass (RP Edit).esp
> Beautiful cities of Morrowind.ESP
> SWP - Foyada Mamaea Overhaul Patch.ESP
> SWP - Caldera Mine Patch.ESP

You'd have to test how LOOT sorts it against.
[Caldera Mine Expanded](https://www.nexusmods.com/morrowind/mods/45194)
[Foyada Mamaea Overhaul](https://www.nexusmods.com/morrowind/mods/46424/)

**Comment by IllusiveMan196** [comment]
Before both Caldera mods, and Foyada Mamaea as well

File: masterlist.yaml [hunk 1]
```diff
@@ -514,6 +514,15 @@ plugins:
...
         condition: 'many("gr_ScriptImprovements.es(m|p)")'
 
+###### Environment, Landscape & Flora ######
+  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
+    after:
+      - name: 'DD_Caldera_Expansion.esp'
+      - name: 'Walkers of Morrowind.esp'
+      - name: 'Foyada Mamaea Overhaul.ESP'
+      - name: 'DD_Caldera_Expansion.esp'
+      - name: 'Order_Of_Sanctity.ESP'
```
Comment (MacSplody) [inline masterlist.yaml]: You only need the `name:` on files if you're adding a condition or display name.
Reordered alphabetically to make it easier to read.

```suggestion
      - 'DD_Caldera_Expansion.esp'
      - 'Foyada Mamaea Overhaul.ESP'
      - 'Order_Of_Sanctity.ESP'
      - 'Walkers of Morrowind.esp'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -514,6 +514,14 @@ plugins:
         subs: [ '"Script Improvements" version' ]
         condition: 'many("gr_ScriptImprovements.es(m|p)")'
 
+###### Environment, Landscape & Flora ######
+  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
```
Comment (MacSplody) [inline masterlist.yaml:518]: Add the location as well.
```suggestion
  - name: 'Better Landscapes Stonewood Pass (RP Edit).esp'
    url: [ 'https://www.nexusmods.com/morrowind/mods/49464/' ]
```

**Comment by MacSplody** [comment]
@IllusiveMan196 Thanks.

**Pull request merged**